copyright-header.config.json

```diff
@@ -0,0 +1,48 @@
+{
+    "license": "copyright-header.txt",
+    "ignore": [
+        "**/*.svg",
+        "**/*.scss.d.ts",
+        "**/.DS_Store",
+        "**/*.gitignore",
+        ".gitattribute",
+        "yarn.lock",
+        ".prettierignore",
+        ".yarnrc",
+        ".gitattributes",
+        "Dockerfile",
```

Comment:
Dockerfile is interesting enough that it probably should be given a copyright header

Reply:
It has a copyright header but the checker cannot check for files without a type; I will have to check what it accepts for files which doesn't have a extension